prow/config/config_test.go

```diff
@@ -8263,6 +8271,7 @@ deck:
 default_job_timeout: 24h0m0s
 gangway: {}
 gerrit:
+  AllowedPresubmitTriggerRe: {}
```

Comment:
This should not be included, please fix the json marshalling annotations.

Reply:
Removed.